src/pytopmod/core/dcel/primitives.py

```diff
@@ -0,0 +1,67 @@
+"""Convenience functions to create primitive DCEL Meshes."""
+from pytopmod.core.dcel import mesh as dcel_mesh
+
+
+def tetrahedron() -> dcel_mesh.Mesh:
+    mesh = dcel_mesh.Mesh()
+
+    v_1, v_2, v_3, v_4 = (
+        mesh.create_vertex((1.0, 1.0, 1.0)),
+        mesh.create_vertex((1.0, -1.0, -1.0)),
+        mesh.create_vertex((-1.0, 1.0, -1.0)),
+        mesh.create_vertex((-1.0, -1.0, 1.0)),
+    )
+
+    f_1, f_2, f_3, f_4 = (
+        mesh.create_face(),
+        mesh.create_face(),
+        mesh.create_face(),
+        mesh.create_face(),
+    )
+
+    mesh.create_edge(v_2, v_1, f_4, f_1, "e6", "e3")
```

Comment:
Why do we need to hard-code e6 and e3 ? 
create_vertex and create_face both go to the base mesh class, right ? 
Where are the edges e6 and e3 are created ?

Reply:
We are doing this at the moment because I haven't yet implemented support for half-edges in DCEL, i.e support for null values for vertex_2, face_2 and vertex_2_next.
The edges e6 and e3 are created by the 3rd and 6th calls to `mesh.create_edge()`, so before they exist I am referring to them using their future key, which I agree is not ideal.
The proper way to do this would be to construct the mesh from scratch with `create_point_sphere()` (not yet implemented) and using `operators.insert_edge()` operations like in DLFL, to load from a dedicated file format (e.g an array representation of the edge_nodes + vertex positions and do some pointer chasing, or import an OBJ file directly, but I haven't implemented `obj_to_mesh` for DCEL yet.

Reply:
Oh wait, we are referencing future edges, lol, I missed that completely.
Yeah, I agree that it is not the best way, but it is fine until we get insert_edge.

Ideally even reading a mesh or creating a primitive should use only manifold preserving operations, but for perf reasons we can shortcut.